apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -519,10 +557,19 @@ public String completeMultipartUpload(MultipartUpload mpu, List<MultipartPart> p
          String path = mpu.containerName() + "/" + getMPUPartName(mpu, part.partNumber());
          builder.add(Segment.builder().path(path).etag(part.partETag()).sizeBytes(part.partSize()).build());
       }
-
       return api.getStaticLargeObjectApi(regionId, mpu.containerName()).replaceManifest(mpu.blobName(),
             builder.build(), mpu.blobMetadata().getUserMetadata(), getContentMetadataForManifest(mpu.blobMetadata().getContentMetadata()));
    }
+   
+   public String completeMultipartUploadDLO(MultipartUpload mpu, List<MultipartPart> parts) {
```

Comment:
Remove - we should not expose dynamic large objects in the portable abstraction!  Users can call the provider-level APIs for this functionality.